pages/courses/index.js

```diff
@@ -11,10 +11,10 @@ function Courses({ allCourses }) {
         <title>Lista de Bootcamps - Bootcamp Web3Dev</title>
       </Head>
       <div className="max-w-7xl mx-auto py-2 px-6">
-        <h4 className="mb-6 text-2xl font-bold text-black-300 dark:text-white-100">
-          Bootcamps
-        </h4>
-        <ul className="grid grid-flow-row md:grid-cols-3 gap-8 overflow-hidden p-2 grid-cols-1">
+        <h1 className="mb-6 text-2xl font-bold text-black-300 dark:text-white-100">
+          Cursos
```

Comment:
Is reviewed and correct the option to call it Cursos instead of bootcamps? If so, then we need to change the title on line 11 to list of Courses, so it matches the heading.

Reply:
Thanks for feedback ! We realy changing name bootcamps to courses ! In the future maybe call Projects !